content/08-cardano-sidechains/03-example-evm-sidechain/00-testnet-disclaimer.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Testnet disclaimer
+metaTitle: Testnet disclaimer
+---
+
+By using the example EVM sidechain testnet, you understand the sidechain is in
+development and that use of the example EVM sidechain testnet is entirely at your own
+risk.  
+
+You also acknowledge and agree to have an adequate understanding of the risks
+associated with use of the example EVM sidechain testnet and that all information and
+materials published, distributed or otherwise made available on Cardano Docs is
+provided for non-commercial, personal use, and testing purposes only. This includes
+FUEL tokens which have no economic value and are provided only for the purpose of
```

Comment:
test currency tokens which have no economic value and are provided only for the purpose of